enterprise/frontend/src/metabase-enterprise/whitelabel/components/ImageToggle/ImageToggle.tsx

```diff
@@ -30,17 +30,13 @@ const ImageToggle = ({
       <ImageContainer>{children}</ImageContainer>
       <ToggleContainer>
         <ToggleLabel htmlFor={toggleId}>{label}</ToggleLabel>
-        <Toggle
+        <Switch
```

Comment:
This is unrelated to the main change (new settings), but it changes the Metabot toggle to use the new component from `metabase/ui`
![image](https://github.com/metabase/metabase/assets/1937582/57f93c76-0dab-4833-ba58-10753991ef77)